nodejs-app/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM alpine:3.10  AS base
```

Comment:
For build use node js alpine image (node:10.16.3-alpine ), to reduce the time for build the artifact. Example -as you are using in to release stage  node:10.16.3-alpine  AS release